### ISSUE TITLE
Support For "In-Container" Mode

### ISSUE THREAD

**Issue by xVanTuring** [naver/egjs-infinitegrid#249]
## Description
Seems that infinitegrid did not support `In-Container` Mode, it won't load any more when scroll to the bottom.
![default](https://user-images.githubusercontent.com/11146248/49297218-32b9c380-f4f5-11e8-9d81-e4a2c1fe98f5.png)

## Steps to check or reproduce
[codepen](https://codepen.io/xVanTuring/pen/bQOMbm)

The demo won't load any more when scroll to the bottom.
## Extra 
 It's caused by the code in `DOMRender`, make the default `"container"` to be the window
``` js
_init(el) {
// ***
 if(/**/){
 }
 else{
    this.view = window;
    this.container = element;
 }
}
```
### Suggest
So I suggest that add a test for parentNode className with `eg-container` or whatever, to use the parentNode to be the `"container"`
``` js
if (element.parentNode.className.indexOf("eg-container") !== -1) {
   this.view = element.parentNode;
} else {
  this.view = window;
}
this.container = element;
```
> And I test it on my local machine, it works.

Hope this is useful. 😀

**Comment by younkue** [comment]
@xVanTuring  Is this demo you want?

https://codepen.io/daybrush/pen/eQXBoe

```
var ig = new eg.InfiniteGrid(".eg-container",{
  isOverflowScroll: document.querySelector(".contents"),
});
```

It does not yet provide a container option, but it is available via the temporary option isOverflowScroll. This problem will be resolved in 3.6.0.

**Comment by xVanTuring** [comment]
YEAH! 😱 All thanks!
